lib/src/components/NxCombobox/__tests__/NxCombobox.test.tsx

```diff
@@ -83,18 +83,122 @@ describe('NxCombobox', function() {
         expect(inputElement).toHaveValue('Foo');
       });
 
-  it('sets aria-expanded on the input to true when focused',
+  it('sets aria-expanded to true on the input when initially focused and there are matches', async function() {
+    const { getByRole } = quickRender({ matches: [{ id: '1', displayName: 'Foo' }] }),
+        inputElement = getByRole('combobox');
+
+    expect(inputElement).toHaveAttribute('aria-expanded', 'false');
+
+    // initial focus of the input
+    inputElement.focus();
+    expect(inputElement).toHaveAttribute('aria-expanded', 'true');
+
+  // visibility of dropdown when focused is tested in visual tests
+  });
+
+  it('set aria-expanded to false on the input if there are no matches, even if focused', async function() {
+    const { rerender, getByRole } = quickRender(),
+        inputElement = getByRole('combobox');
+
+    expect(inputElement).toHaveAttribute('aria-expanded', 'false');
+
+    inputElement.focus();
+    expect(inputElement).toHaveAttribute('aria-expanded', 'false');
+
+    rerender(<NxCombobox {...minimalProps} value= {'foo'}/>);
+    expect(inputElement).toHaveAttribute('aria-expanded', 'false');
```

Comment:
Shouldn't this one be true? Shouldn't this show the loading alert?

Reply:
> Shouldn't this one be true? Shouldn't this show the empty alert?

Aria-expanded was previously set according to `showDropdown && inputIsFocused`, and now `showDropdown`, so without matches, `aria-expanded` is set to false.
I just read through w3.org, and it says "when the combobox popup is not visible, the element with role combobox has aria-expanded set to false".  
Above that it mentions "the popup is an element that has a role listbox, tree, grid or dialog" (https://www.w3.org/WAI/ARIA/apg/patterns/combobox/#:~:text=The-,popup,-is%20an%20element). If the empty message div has a parent element of `role="alert"`,  I think we should leave it as `aria-expanded= false`?

Reply:
> Aria-expanded was previously set according to showDropdown && inputIsFocused, and now showDropdown, so without matches, aria-expanded is set to false.

Ah you're right.  In the previous impl it was false in that circumstance, and we aren't looking to change that